### DISCUSSION TITLE
IOS/Android touchscreens support?

### DISCUSSION THREAD

**Discussion by pradel** [remirror/remirror#798] [Q&A]
Hey, just came across this project which looks super cool!
I am using Slate for the editor in my app, writing on mobile or tablet is really buggy. I was wondering what is the ios/android support and does this editor works well with touchscreens in general? I was not able to find this information on the docs so asking here :)

**Comment by ghost** [comment]
Currently remirror does not have any support for IOS/Andriod. In future, remirror is going to get integrated in react-native. https://github.com/remirror/remirror/issues/37

**Reply by pradel** [reply]
Thanks for the reply but I am not talking about support in react-native, rather support in a mobile browser when visiting a website.

**Reply by ghost** [reply]
Yes. It works in mobile browser. I don’t know if all extensions are supported in browser. You need to test it.

**Reply by pradel** [reply]
Okay, I was asking because I can't test on IOS and on android tabled because I don't own such devices :)

**Reply by onurozkan** [reply]
@pradel 

You can sign up for a free trial from browserstack.

**Reply by pradel** [reply]
@onurozkan that's a good idea, thanks for the suggestion :)
